pkg/handler/project/testdata/get_project/200.json

```
@@ -0,0 +1,40 @@
+{
+	"data": {
+		"id": "8dc3be2e-19a4-4942-8a79-56db391a0b15",
+		"createdAt": "",
+		"updatedAt": "",
+		"name": "Fortress",
+		"type": "dwarves",
+		"status": "active",
+		"startDate": "2022-11-01T00:00:00Z",
+		"endDate": null,
```

missing Client Email, Project email, Country, and Industry fields. Until now, we don't have data and design for them. So leave these fields with TBD value:

```suggestion
		"endDate": null,
		"projectEmail": "TBD",
		"clientEmail": "TBD",
		"industry": "TBD",
		"country": "TBD",
```